sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_choice_screen_engagement_v1/schema.yaml

```diff
@@ -0,0 +1,51 @@
+fields:
+
+- mode: NULLABLE
+  name: date
+  type: DATETIME
+  description:
```

Comment:
might want to add some descriptions here

Reply:
done! The plan was to add it at the very end once everything else was in place.